crates/networking/discv5/src/discovery.rs

```diff
@@ -129,21 +137,46 @@ impl Discovery {
         self.find_peer_active = true;
 
         let query = match subnet_id {
-            Some(id) => QueryType::FindSubnetPeers(vec![Subnet::Attestation(id)]),
-            None => QueryType::FindPeers,
+            Some(id) => QueryType::SubnetPeers(vec![Subnet::Attestation(id)]),
+            None => QueryType::Peers,
+        };
+
+        self.start_query(query, target_peers);
+    }
+
+    pub fn discover_sync_committee_peers(&mut self, target_peers: usize, subnet_id: Option<u8>) {
```

Comment:
This function isn't used currently could it be removed for now and added back when it is used